src/types/common.ts

```diff
@@ -117,3 +117,15 @@ export type WalletAddress = string;
 export const DeadAdresses: Array<WalletAddress> = [
   '0xdeaddeaddeaddeaddeaddeaddeaddeaddeaddead',
 ];
+
+export enum EHttpMethods {
```

Comment:
```suggestion
export enum EventHttpMethods {
```

Maybe just spell it out, unless `E` doesn't actually stand for `Event` 😬 .
Or `TrackingEventHttpMethods` or `HeapHttpMethods` 🤷🏻‍♂

Reply:
Doesnt stand for Event. Stands for Enum HttpMethods, when importing it has a clear indication that these are enums vs an Interface. I could have an Interface with the same name and same description. EHttpMethods as this enum with the strings and an interface as IHttpMethods as another declared custom type.

Reply:
Gotcha.  More common practice would be to just have a singular Enum them,  so `HttpMethod`